test/binomial.jl

```diff
@@ -1,6 +1,3 @@
-using Distributions
-using Base.Test
```

Comment:
It's quite convenient that a test file can be run in isolation so I think we should keep these.